test/e2e/upgradetest/upgrade_test.go

```diff
@@ -43,7 +48,6 @@ func TestResize(t *testing.T) {
 		if err := e2eutil.DeleteCluster(t, testF.KubeCli, testClus, &e2eutil.StorageCheckerOptions{}); err != nil {
 			t.Fatal(err)
 		}
-		time.Sleep(10 * time.Second)
```

Comment:
I thought that `DeleteCluster()` already checks that all resources are deleted by calling `waitResourcesDeleted()` and `waitBackupDeleted()` which waits for the pods, services, backup side car and storage to be deleted. 
Why is the extra sleep needed? Perhaps for the TPR object to be deleted?